docs/guides/migrating-from-camunda-7/technical-details.md

```diff
@@ -132,12 +132,10 @@ The following attributes/elements can be migrated:
 
 The following attributes/elements **cannot** yet be migrated:
 
-- Form handling:
-  - `camunda:formHandlerClass`
-  - `camunda:formData`
-  - `camunda:formProperty`
-  - `camunda:formRefVersion`
-- `camunda:taskListener`
```

Comment:
@jonathanlukas Should we add this item to the section above, i.e. 'The following attributes/elements can be migrated'?

Reply:
Good catch, I added it above :)